Plugins/Flow.Launcher.Plugin.Explorer/Search/SearchManager.cs

```diff
@@ -39,19 +39,23 @@ internal List<Result> Search(Query query)
             if (quickFolderLinks.Count > 0)
                 return quickFolderLinks;
 
-            if (string.IsNullOrEmpty(query.Search))
+            if (string.IsNullOrEmpty(querySearch))
                 return results;
 
-            if (!FilesFolders.IsLocationPathString(querySearch))
-                return WindowsIndexFilesAndFoldersSearch(query, querySearch);
-
-            var locationPath = query.Search;
+            var isEnvironmentVariable = EnvironmentVariables.IsEnvironmentVariableSearch(querySearch);
 
-            if (EnvironmentVariables.IsEnvironmentVariableSearch(locationPath))
-                return EnvironmentVariables.GetEnvironmentStringPathSuggestions(locationPath, query, context);
+            if (isEnvironmentVariable)
+                return EnvironmentVariables.GetEnvironmentStringPathSuggestions(querySearch, query, context);
 
             // Query is a location path with a full environment variable, eg. %appdata%\somefolder\
-            if (locationPath.Substring(1).Contains("%"))
+            var isEnvironmentVariablePath = querySearch.Substring(1).Contains("%\\");
+
+            if (!FilesFolders.IsLocationPathString(querySearch) && !isEnvironmentVariable && !isEnvironmentVariablePath)
```

Comment:
nitpick: the `!isEnvironmentVariable` is redundant here. Otherwise LGTM...

Reply:
Good pick up, thank you